src/components/Club/Home/index.js

```diff
@@ -1,18 +1,18 @@
-import styles from 'styles/Club/Home/Common/frame.module.scss';
-import SideBar from './SideBar';
-import ClubIntro from '../Intro/ClubIntro';
-import Activities from '../Activities/Activities';
-import Review from '../Review/Review';
-import Calendar from '../Calendar';
-import ClubNotice from '../Notice/ClubNotice';
 import { useCallback, useEffect, useState } from 'react';
-import Apply from '../Apply/Apply';
+import { getClubInfo } from 'redux/slices/clubhome';
 import { useRouter } from 'next/router';
 import { useDispatch, useSelector } from 'react-redux';
-import { getClubInfo } from 'redux/slices/clubhome';
 import { changeComp } from 'redux/slices/chageComp';
+import ClubIntro from './Intro/ClubIntro';
+import ClubNotice from './Notice/ClubNotice';
+import Activities from './Activities/Activities';
+import Calendar from './Calendar';
+import Review from './Review/Review';
```

Comment:
```suggestion
import Activities from './Activities';
import Calendar from './Calendar';
import Review from './Review';
```
파일명 index 로 바꾸면 이렇게 가능하지 않을까여 빨리 해보고 알려주세연

Reply:
그건 이제 차차 해나갈겁니다